scene2d/src/main/kotlin/ktx/scene2d/factory.kt

```diff
@@ -219,12 +219,16 @@ inline fun <S> KWidget<S>.horizontalGroup(
 @Scene2dDsl
 @OptIn(ExperimentalContracts::class)
 inline fun <S> KWidget<S>.image(
-  drawableName: String,
+  drawableName: String? = null,
```

Comment:
Doesn't it leave 2 `image` methods that potentially have no parameters? If both have essentially the same effect, maybe let's leave the one with the drawable name as is, and make the `Drawable` argument nullable in the other one alone?

Reply:
Will check that. I had a similar concern about JVM that we end up with two methods of the same signature but I think that's not the case because on method takes 3 arguments and the other one 2. Not quite sure what kind of methods Kotlin compiler is creating with the default null argument but I think it should be fine, otherwise the IDE most likely would have shown me an alert 😅

Anyway, I will adapt it to your suggestion. That should work as well.